### DISCUSSION TITLE
How can I get more information when Highs_passModel() returns "Warn"?

### DISCUSSION THREAD

**Discussion by benjaminplong** [ERGO-Code/HiGHS#1608] [Q&A] [answered]
I am attempting to solve a knapsack problem and the problem is failing to initialize. The call to Highs_passModel returns a "Warn" status, but if I ignore the warning and continue with the solve the expected goal value is found.

Is there a way to get more information as to why the warning status is returned?

**Comment by jajhall** [comment]
How are you calling HiGHS? Directly from C?

The default logging messages from HiGHS will indicate the reason for the warning. It's most likely to be due to you passing small matrix entries that are ignored

**Reply by benjaminplong** [reply]
Ultimately through a compiled .so library linked through Java JNA. I think I was able to find the output after updating my code to ignore the warning in all the right places. 

"WARNING: LP matrix packed vector contains 4 |values| in [0, 0] less than or equal to 1e-09: ignored"

And I was able to find 0s in the test data which was getting set as coefficients in a constraint. Skipping those coefficients fixed it

**Reply by jajhall** [reply, marked as answer]
So, you're passing explicit zeros in your definition of the matrix entries. Even though nothing is lost by ignoring them, it's still worth warning people that they are doing this.
